package.json

```diff
@@ -9,6 +9,8 @@
         "@livechat/customer-sdk": "^3.1.0",
         "@loadable/component": "^5.15.2",
         "@svgr/webpack": "^6.2.1",
+        "@tanstack/react-table": "^8.7.4",
+        "@u-wave/react-vimeo": "^0.9.8",
```

Comment:
We don't need this too

Reply:
hi @habib-deriv we need this one `@tanstack/react-table": "^8.7.4",`

Reply:
Is this "@u-wave/react-vimeo": "^0.9.8", needed?